src/infra/alarm/template.yaml

```diff
@@ -667,13 +668,13 @@ Resources:
       ActionsEnabled: true
       AlarmActions:
         - !Ref AlertingSNSHighAlertNotificationTopicARN
-      AlarmDescription: "An error occured with the query to DynamoDB"
+      AlarmDescription: "An error occurred with the query to DynamoDB once every minutes for two consecutive periods"
```

Comment:
@chiarapascucci-gds did you mention removing this? Missed what we decided on.

Reply:
hey @domantas-gds @MinaMahdi - not removing, just changing the code so that this doesn't possible get logged alongside the no response alarm (which needs to be created for the status retriever function). So we still need this

Reply:
as part of this PR, could we please add this alarm (please revise the threshold as appropriate):

`  DBQueryNoResponseError:
    Type: AWS::CloudWatch::Alarm
    Properties:
      ActionsEnabled: true
      AlarmActions:
        - !Ref AlertingSNSLowAlertNotificationTopicARN
      AlarmDescription: "DynamoDB query no response error"
      ComparisonOperator: GreaterThanOrEqualToThreshold
      Threshold: 1
      EvaluationPeriods: 1
      TreatMissingData: notBreaching
      MetricName: DB_QUERY_ERROR_NO_RESPONSE
      Namespace: !Ref SystemTagValue
      Dimensions:
        - Name: service
          Value: StatusRetrieverFunction
      Period: 60
      Statistic: Sum
      Unit: Count`

this is because this metric (DB_QUERY_ERROR_NO_RESPONSE) can be logged by the Status Retriever Function too, but I do not think we have an alarm for this case?

Reply:
Yep, no alarm for it  - added, thank you 🙏